bigquery/quickstart.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigquery_quickstart]
+# Imports the Google Cloud client library
+require "google/cloud"
+
+# Your Google Cloud Platform project ID
+project_id = "YOUR_PROJECT_ID"
+
+# Instantiates a client
+gcloud = Google::Cloud.new project_id
```

Comment:
@jmdobry Is it critically important for every quickstart to have variables suffixed with `_client`?

No other Ruby samples use this, including google-cloud-ruby quickstarts.  I would like the quickstart code to be conventional with code seen everywhere else.

For example, google-cloud-ruby has a quickstart on its homepage at: https://googlecloudplatform.github.io/google-cloud-ruby/#/

```ruby
require "google/cloud"

gcloud = Google::Cloud.new "my-todo-project-id",
                           "/path/to/keyfile.json"
datastore = gcloud.datastore

product = datastore.find "Product", 123
```

And all other google-cloud-ruby snippets follow the following template:

```ruby
require "google/cloud"

gcloud = Google::Cloud.new
storage = gcloud.storage

all_buckets = storage.buckets
```